modules/olmv1-installing-an-operator.adoc

```diff
@@ -168,16 +172,20 @@ items:
   metadata:
     annotations:
       kubectl.kubernetes.io/last-applied-configuration: |
-        {"apiVersion":"olm.operatorframework.io/v1alpha1","kind":"ClusterExtension","metadata":{"annotations":{},"name":"pipelines-operator"},"spec":{"channel":"latest","installNamespace":"openshift-operators","packageName":"openshift-pipelines-operator-rh","pollInterval":"30m"}}
+        {"apiVersion":"olm.operatorframework.io/v1alpha1","kind":"ClusterExtension","metadata":{"annotations":{},"name":"pipelines-operator"},"spec":{"channel":"latest","installNamespace":"pipelines","packageName":"openshift-pipelines-operator-rh","serviceAccount":{"name":"pipelines-installer"},"pollInterval":"30m"}}
```

Comment:
Do you think there is any value to making this block source,yaml to get the syntax highlighting (tho I think docs.rh might not really highlight much, idr)

Reply:
I've used `text` throughout, and I have debated back and forth on this. I'll add it to my OLM v1 GA tracking ticket for me and Alex to discuss for the push out of tech preview.

Reply:
`source,yaml` would definitely be more legible.